source/clear-linux/tutorials/telemetry-backend/Telemetry-backend.rst

```diff
@@ -0,0 +1,495 @@
+   INFO  [alembic.runtime.migration] Will assume transactional DDL.
+   INFO  [alembic.runtime.migration] Running upgrade  -> 3230c615d6e0, empty message
+   INFO  [alembic.runtime.migration] Running upgrade 3230c615d6e0 -> 466cf2f35d67, empty message
+
+   Install complete (installation folder: /var/www/telemetry)
+
+Once the installation is complete you can use your web browser and view the
+new server by opening the web browser on the system you installed the backend
+server onto and type in ``localhost`` in the address bar.  You should see a
+web page similar to the one shown in figure 1:
+
+.. figure:: figures/telemetry-backend-1.png
+   :scale: 50 %
+   :alt: Telemetry UI
+
+   Figure 1: :guilabel:`Telemetry UI`
+
+
+Redirect telemetry records
+==========================
+
+Telemetry records from your system are sent to the server location defined in
+the :file:`/usr/share/defaults/telemetrics/telemetrics.conf` configuration
+file. You can customize this by copying this file to
+:file:`/etc/telemetrics/telemetrics.conf` and changing the ``server=``
+setting to your new server location.
+
+#. Create the :file:`/etc/telemetrics` directory and make it your current
+   working directory.
+
+   .. code-block:: console
+
+      sudo mkdir -p /etc/telemetrics
+      cd /etc/telemetrics
+
+
+#. Copy the default :file:`telemetrics.conf` file to the new
+   :file:`/etc/telemetrics` directory.
+
+   .. code-block:: console
+
+      sudo cp /usr/share/defaults/telemetrics/telemetrics.conf .
+
+#. Edit the new :file:`/etc/telemetrics/telemetrics.conf` file with your
+   editor using the :command:`sudo` directive and change the
+   :guilabel:`server=` setting to ``http://localhost/v2/collector`` and save
+   this change in the new file.
+
+   .. code-block:: console
+
+      server=http://localhost/v2/collector
+
+   You can also use the fully qualified domain name for your server instead of
+   :guilabel:`localhost`.
+
+#. Restart the :command:`telemd` daemon to reload the configuration file.
+
+   .. code-block:: console
+
+      systemctl restart telemd
+
+Test the new telemetry backend server
+=====================================
+
+|CL| includes a telemetry test probe called :command:`hprobe` that will send a
+``hello`` record to the telemetry backend server.  To test that the telemetry
+records are now going to your new destination, run the :command:`hprobe`
+command to send a ``hello`` record to the server as follows:
+
+.. code-block:: console
+
+   hprobe
+
+The record should show up on your new server console as shown in figure 2:
+
+.. figure:: figures/telemetry-backend-2.png
+   :scale: 50 %
+   :alt: Telemetry UI
+
+   Figure 2: :guilabel:`Telemetry UI`
+
+Congratulations!  You've just set up and enabled a new telemetrics backend
+server, redirected the records from your local machine to this new server and
+tested it using the :command:`hprobe` command to send a ``hello`` record to
+it.
+
+Additional resources
```

Comment:
This section contains more than the additional resources. I would title the section "Deployment script usage", preface the code-block with a sentence similar to "The :file:deploy.sh` script should be used in the following manner:"
And then have the links to the resources under a Additional resources heading.

Reply:
I removed the deploy.sh usage message since it is redundant

Reply:
Well done.